src/components/customer/Paid60.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+
+const PaidSixty = () => {
+  const customers = useSelector((state) => state.customers?.data);
+  const data = JSON.parse(localStorage.getItem('user'));
+  const { user } = data || {};
+  const admins = user.role === 'admin' || user.role === 'superadmin';
+
+  console.log(customers);
+
```

Comment:
please kindly remove console logs from your code base, as this will make your code cleaner